.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   linux-tests:
     strategy:
       matrix:
-        node: [10, 12, 14, 16]
+        node: [10, 12, 14, 16, 18]
```

Comment:
Given that we've tentatively merged Node 20 support into PWA Kit @ 3.4, would you consider adding Node 20 here?

Reply:
Do we want to also include that info in the README? https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/580fc611b22f96cb68830ec3e074f600e89e0f73/README.md#L11

Reply:
I've added node 20 to the CI and updated the `README` to include Node `^18.x`.

@adamraya As for the `yarn install` error, this doesn't appear on my machine and the CI seems to be able to run `yarn install` successfully for both node 18 and 20: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/actions/runs/7546319904/job/20543824549

What version of `yarn` do you have installed on your machine? Have you tried deleting `node_modules` and `yarn.lock` and then running `yarn install`?